Cargo.toml

```diff
@@ -11,24 +11,39 @@ documentation = "https://docs.rs/reconnecting-jsonrpsee-ws-client"
 keywords = ["jsonrpc", "json", "websocket"]
 readme = "README.md"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
 [dependencies]
-futures = "0.3"
-jsonrpsee = { version = "0.22", features = ["ws-client"] }
-serde_json = { version = "1", features = ["raw_value"] }
-tokio = { version = "1.35", features = ["rt-multi-thread", "sync"] }
-tokio-stream = "0.1"
-tokio-retry = "0.3"
-tracing = "0.1"
+futures = { version = "0.3", default-features = false }
+jsonrpsee = { version = "0.22", default-features = false }
+serde_json = { version = "1", features = ["raw_value"], default-features = false }
+tokio = { version = "1.35", features = ["sync"], default-features = false }
+tokio-stream = { version = "0.1", default-features = false }
+tracing = { version = "0.1", default-features = false }
 thiserror = "1"
+wasm-bindgen-futures = { version = "0.4.41", optional = true }
+wasm-tokio-retry = { package = "tokio-retry", git = "https://github.com/polytope-labs/rust-tokio-retry", branch = "master", optional = true }
```

Comment:
There are no crates with "retry-futures" that does this without tokio right? 

tokio::time::Instant and std::time::Instant panics on WASM so this makes sense... I would want to publish
this crate on crates.io but for now it's fine.

Reply:
Actually you're right, i found this one https://github.com/softprops/again